packages/cf-component-form/test/__snapshots__/FormFieldset.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`should render 1`] = `
 <fieldset
-  className="cf-cy1e5a"
```

Comment:
I don't understand why my local build fails without this, while CI says otherwise.